pkg/gui/custom_patch_options_panel.go

```diff
@@ -177,11 +177,20 @@ func (gui *Gui) handlePullPatchIntoNewCommit() error {
 		return err
 	}
 
-	return gui.c.WithWaitingStatus(gui.c.Tr.RebasingStatus, func() error {
-		commitIndex := gui.getPatchCommitIndex()
-		gui.c.LogAction(gui.c.Tr.Actions.MovePatchIntoNewCommit)
-		err := gui.git.Patch.PullPatchIntoNewCommit(gui.State.Model.Commits, commitIndex)
-		return gui.helpers.MergeAndRebase.CheckMergeOrRebase(err)
+	commitIndex := gui.getPatchCommitIndex()
+	head_message, _ := gui.git.Commit.GetCommitMessage(gui.State.Model.Commits[commitIndex].FullRefName())
+
+	// TODO: use the commit message panel here
```

Comment:
This will probably have to wait for [this](https://github.com/jesseduffield/lazygit/pull/2390).

Reply:
That PR is merged now @derteta